lib/services/ios-debug-service.ts

```diff
@@ -171,20 +171,20 @@ class IOSDebugService extends DebugServiceBase implements IPlatformDebugService
 			return result;
 		};
 
-		const results = await this.$devicesService.execute(action, this.getCanExecuteAction(debugData.deviceIdentifier));
-		return _.map(results, r => r.result);
+		const deviceActionResult = await this.$devicesService.execute(action, this.getCanExecuteAction(debugData.deviceIdentifier));
+		return deviceActionResult[0].result;
 	}
 
 	private async debugBrkCore(device: Mobile.IiOSDevice, debugData: IDebugData, debugOptions: IDebugOptions): Promise<string> {
 		await this.$iOSSocketRequestExecutor.executeLaunchRequest(device.deviceInfo.identifier, TIMEOUT_SECONDS, TIMEOUT_SECONDS, debugData.applicationIdentifier, debugOptions.debugBrk);
 		return this.wireDebuggerClient(debugData, debugOptions, device);
 	}
 
-	private async deviceStart(debugData: IDebugData, debugOptions: IDebugOptions): Promise<string[]> {
+	private async deviceStart(debugData: IDebugData, debugOptions: IDebugOptions): Promise<string> {
 		await this.$devicesService.initialize({ platform: this.platform, deviceId: debugData.deviceIdentifier });
 		const action = async (device: Mobile.IiOSDevice) => device.isEmulator ? await this.emulatorStart(debugData, debugOptions) : await this.deviceStartCore(device, debugData, debugOptions);
-		const results = await this.$devicesService.execute(action, this.getCanExecuteAction(debugData.deviceIdentifier));
-		return _.map(results, r => r.result);
+		const deviceActionResult = await this.$devicesService.execute(action, this.getCanExecuteAction(debugData.deviceIdentifier));
+		return deviceActionResult[0].result;
```

Comment:
previously this has not been failing when the `deviceActionResult` is `null/undefined` or empty array, but now it will fail with error - `Cannot read property result of undefined.` Is this expected? In case not, you can use:
```TypeScript
return deviceActionResult && deviceActionResult[0] && deviceActionResult[0].result;
```